pskb_website/static/js/editor_utils.js

```diff
@@ -407,7 +407,7 @@ function openFullscreen() {
 function resizeEditor() {
     var lineHeight = editor.renderer.lineHeight;
     var maxLines = document.getElementById('editor-wrapper').offsetHeight / lineHeight;
-    editor.setOption('maxLines', Math.floor(maxLines) - 1);
```

Comment:
Wasn't the maxLines option causing some of the performance issues?  Why was there a `- 1` to begin with? Just curious.